rpcs3/Emu/Cell/Modules/cellSpurs.h

```diff
@@ -811,12 +811,22 @@ struct alignas(128) CellSpursTaskset
 
 	CHECK_SIZE(TaskInfo, 48);
 
-	be_t<v128> running;                          // 0x00
-	be_t<v128> ready;                            // 0x10
-	be_t<v128> pending_ready;                    // 0x20
-	be_t<v128> enabled;                          // 0x30
-	be_t<v128> signalled;                        // 0x40
-	be_t<v128> waiting;                          // 0x50
+	struct atomic_tasks_bitset
+	{
+		atomic_be_t<u32> values[4];
+
+		u32 get_bit(u32 bit) const
+		{
+			return values[bit / 32] & ((1u << 31) >> (bit % 32));
```

Comment:
This seems a little unusual that bit indexes would start from the left.
Normally bit 0 = right-most bit (and this is endian independent).

Reply:
Tell that to Sony.

Reply:
I would have thought we'd already have this mapped into host bit order, such that 1+1 = 2 = 1 << 1
Otherwise we'd have to be doing crazy bit flipping in all math operations (to get 0x80000000 + 0x80000000 = 0x40000000)

Reply:
this is about **cellSpurs** behaviour.